src/code/z_kankyo.c

```diff
@@ -664,46 +701,46 @@ void Environment_Init(PlayState* play2, EnvironmentContext* envCtx, s32 arg2) {
     play->envCtx.precipitation[PRECIP_SNOW_MAX] = 0;
     play->envCtx.precipitation[PRECIP_SOS_MAX] = 0;
 
-    D_801F4E31 = envCtx->skyboxConfig;
+    sInitSkyboxConfig = envCtx->skyboxConfig;
 
-    var_a0 = 0;
+    dayOffset = 0;
     if (((void)0, gSaveContext.save.day) != 0) {
-        var_a0 = ((void)0, gSaveContext.save.day) - 1;
+        dayOffset = ((void)0, gSaveContext.save.day) - 1;
```

Comment:
Took a look at the codebase and noticed there *already is* one:
```c
#define CURRENT_DAY (((void)0, gSaveContext.save.day) % 5)
```
So maybe it doesn't make sense to define another one, at least for right now. It looks like it's just kankyo that uses `(void)0, gSaveContext.save.day)` by itself without using `CURRENT_DAY`. It's probably out-of-scope to introduce another macro in this PR, so you can feel free to ignore my original comment.

Reply:
Yeah, distinguishing between `(void)0, gSaveContext.save.day` and `(((void)0, gSaveContext.save.day) % 5)` is the reason I avoided that, all I can think of is `CURRENT_DAY_ALT` for `(void)0, gSaveContext.save.day` since it's much less common

Reply:
Yeah, no need to concern yourself with it in this PR. It's something to think about for whoever does more work on kankyo next.